util/src/timestamp.rs

```diff
@@ -1,22 +1,106 @@
-use chrono::prelude::*;
+use chrono::{prelude::*, TimeZone};
+use lazy_static::lazy_static;
+use ntp;
+use serde::{Deserialize, Serialize};
+use std::sync::RwLock;
 use std::time::Duration;
 
-/// Function to get timestamp from system as UTC Unix timestamp, seconds since Unix epoch
-pub fn get_timestamp() -> i64 {
+/// NTP Timestamp information
+#[derive(Debug, Default, Serialize, Deserialize, PartialEq, Clone)]
+pub struct TimestampNTP {
+    /// Difference between NTP and system timestamp
+    pub ntp_diff: Duration,
+    /// Flag to indicate if NTP is bigger or smaller
+    /// than system timestamp
+    pub bigger: bool,
+    /// Address to require NTP timestamp information
+    pub server: String,
```

Comment:
Maybe we can remove this field from here and add `server` as a parameter to `update_global_timestamp`?